build_runner/lib/src/generate/phase.dart

```diff
@@ -53,7 +70,7 @@ class BuildAction {
   ///
   /// [hideOutput] specifies that the generated asses should be placed in the
   /// build cache rather than the source tree.
-  factory BuildAction(
+  factory BuilderBuildAction(
```

Comment:
Maybe `BuilderAction` to match `PostProcessBuildAction` vs `PostProcessBuilderAction`?

Or maybe `InBuildAction` and then `PostBuildPhase` and `PostBuildAction`?